DHIS2/dutyday/logger.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+#this file must be placed in the instances
+set +x
+
+monitlogger() {
+  echo ""
+	echo "----------monitlog----------"
+	echo
+	sudo monit status | awk '/Process/{p=$0} /Not monitored/{print p "\n" $0 "\n"}'
+	echo
+}
```

Comment:
The value of "monit" field could be used here. That way, if we needed to change the command there would be no need to redeploy logger.sh.

In the other hand that's probably unlikely.